includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -0,0 +1,296 @@
+<?php
+
+class DesignSystemGlobalNavigationModel extends WikiaModel {
+	const DEFAULT_LANG = 'en';
+
+	private $wikiId;
+	private $lang;
+
+	public function __construct( $wikiId, $lang = self::DEFAULT_LANG ) {
+		parent::__construct();
+
+		$this->wikiId = $wikiId;
+		$this->lang = $lang;
+	}
+
+	public function getData() {
+		global $wgUser;
+
+		$data = [
+			'logo' => [
+				'links' => [
+					[
+						'type' => 'link-image',
+						'href' => $this->getHref( 'fandom-logo' ),
+						'image' => 'wds-company-logo-fandom-powered-by-wikia',
+						'title' => [
+							'type' => 'text',
+							'value' => 'Fandom powered by Wikia'
+						]
+					]
+				]
+			],
+			'search' => [
+				'module' => [
+					'type' => 'search',
+					'results' => [
+						'url' => $this->getPageUrl( 'Search', NS_SPECIAL, 'fulltext=Search' ),
+						'param-name' => 'query'
+					],
+					'suggestions' => [
+						'url' => WikiFactory::getHostById( $this->wikiId ) . '/index.php?action=ajax&rs=getLinkSuggest&format=json',
+						'param-name' => 'query'
+					],
+					'placeholder-inactive' => [
+						'type' => 'translatable-text',
+						'key' => 'global-navigation-search-placeholder-inactive'
+					],
+					'placeholder-active' => [
+						'type' => 'translatable-text',
+						'key' => WikiaPageType::isCorporatePage()
+							? 'global-navigation-search-placeholder-wikis'
+							: 'global-navigation-search-placeholder-in-wiki'
+					]
+				]
+			],
+			'create_wiki' => [
+				'links' => [
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-navigation-create-wiki-link-start-wikia'
+						],
+						'href' => $this->getHref( 'create-new-wiki' ),
+					]
+				]
+			]
+		];
+
+		if ( $this->lang === static::DEFAULT_LANG ) {
+			$data[ 'fandom_overview' ] = $this->getVerticalsSection();
+			$data[ 'wikis' ] = [
+				'header' => [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "header" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)